package.json

```diff
@@ -35,6 +35,7 @@
     "@storybook/addon-actions": "^5.3.18",
```

Comment:
mostly dev dependencies but there seems to be a lot here are they all needed to support the example app?  if so that is fine.  Noticing `react-scripts` which doesn't appear to be used, stuff for cypress, etc